packages/web-extension/src/observableWallet/util.ts

```diff
@@ -38,7 +38,15 @@ export const outputBuilderProperties: RemoteApiProperties<OutputBuilder> = {
   }
 };
 
-export const txBuilderProperties: RemoteApiProperties<Omit<TxBuilder, 'customize'>> = {
+export const txBuilderProperties: RemoteApiProperties<Omit<TxBuilder, 'customize' | 'buildToken' | 'buildPolicy'>> = {
```

Comment:
@mkazlauskas , I excluded the policy/token build utilities from the TxBuilder remote api interface to avoid cluttering.
I believe this might have been the intent in the original PR, but it was done by excluding these methods from the TxBuilder interface. I added them in the interface, but excluded them from the exposed properties.
Do you agree?

Reply:
Problem with this approach is that ObservableWallet.createTxBuilder returns a TxBuilder so those methods would appear as available but just fail. It would be better to make it actually compatible with messaging by following the pattern of OutputBuilder (`build` method async).